zeebe/gateway/src/main/java/io/camunda/zeebe/gateway/impl/broker/request/BrokerAuthorizationPatchRequest.java

```diff
@@ -18,30 +18,30 @@
 import java.util.List;
 import org.agrona.DirectBuffer;
 
-public class BrokerAuthorizationCreateRequest extends BrokerExecuteCommand<AuthorizationRecord> {
+public class BrokerAuthorizationPatchRequest extends BrokerExecuteCommand<AuthorizationRecord> {
```

Comment:
I considered this, but we can reuse this same request class for removing permissions as it uses the same endpoint. I figured I'd keep it simple and call it a patch request, instead of having 2 separate classes.